pkg/publicshare/manager/json/json.go

```diff
@@ -140,6 +139,7 @@ type commonConfig struct {
 	JanitorRunInterval             int    `mapstructure:"janitor_run_interval"`
 	EnableExpiredSharesCleanup     bool   `mapstructure:"enable_expired_shares_cleanup"`
 	WriteableShareMustHavePassword bool   `mapstructure:"writeable_share_must_have_password"`
```

Comment:
Do we need to rename it to 
```go
PublicWriteableShareMustHavePassword bool `mapstructure:"public_writeable_share_must_have_password"`
 ?

Reply:
we can completely remove it. No checks anymore in the share manager. We now do that in the publicshareprovider.